terraform-e2e/README.md

```diff
@@ -0,0 +1,54 @@
+# terraform-e2e
+
+This directory contains terraform configuration to be used for deploying key servers with terraform, while being able to reuse a project to deploy repeatedly.
```

Comment:
Should you mention this is mostly for testing?

Reply:
Yep, done